includes/dashboard/FurtherUserOptions.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/* 
+ * this is to view users category wise
+ */
+session_start();
+
+if( !( isset( $_SESSION['Username'] ) && isset($_SESSION['Name']) && $_SESSION['Username'] === 'admin' ) )
+{
+	echo "To access this page you need to login as Admin. Please log in first.";
+	die();
+}
+
+
+require_once "../../LocalSettings.php";
+require_once "../Globals.php";
+
+if ( !(isset($_POST['tab']) && isset($_POST['levelId'])) )
+{
+	echo "Sorry, there is some problem<br>";
+	die ();
+}
+
+
+
+$tab = $_POST['tab'];
+$levId = $_POST['levelId'];
+
+
+$sqlConn = new mysqli( $eorgDBserver , $eorgDBuser , $eorgDBpasswd , $catDB );
+
+if( $sqlConn->connect_errno ) 
```

Comment:
__[code_sniffer]__
* Expected 1 space after IF keyword; 0 found
* Expected 0 spaces after opening bracket; 1 found
* Expected 0 spaces before closing bracket; 1 found
* Expected 1 space after closing parenthesis; found 2
* Whitespace found at end of line

<!-- Comment by [SideCI](https://sideci.com) -->